scripts/generate_gallery.py

```diff
@@ -58,8 +58,9 @@
     "introduction": "Introduction",
     "rewrites": "Graph Rewriting",
     "autodiff": "Automatic Differentiation",
-    "scan": "Looping in Pytensor",
-    "optimize": "Optimization in Pytensor",
+    "scan": "Looping in PyTensor",
+    "optimize": "Optimization in PyTensor",
+    "applications": "Applications",
```

Comment:
Maybe "Case Studies" or "Applied Examples" as the user-facing name? No strong preference.

Reply:
Changed in https://github.com/pymc-devs/pytensor/pull/1588/commits/ae9b92527ac4d465961e13a75d6ea3075b151b9c 👍